src/core/cm/launcher/instancemanager.cpp

```diff
@@ -35,6 +36,10 @@ Error InstanceManager::Start()
         return err;
     }
 
+    if (auto err = mGIDPool.Init(allUIDsValid); !err.IsNone()) {
```

Comment:
Validator for GID and UID should be passed as parameter for launcher. It is platform specific part.